packages/odie-client/src/query/index.ts

```diff
@@ -331,7 +332,7 @@ export const useOdieSendMessageFeedback = (): UseMutationResult<
 				return {
 					...currentChatCache,
 					messages: currentChatCache.messages.map( ( m ) =>
-						m.internal_message_id === message.internal_message_id ? { ...m, rating_value } : m
+						m.message_id === message.message_id ? { ...m, rating_value } : m
```

Comment:
When I enabled the staleTime in the cache, I realized that the internal_message_id was not set always. So the feedback wasn't included when loading from React Query Cache

Reply:
Also, to explain a bit more.

When we do a get for a chat, we assign a key to it. That key is used in React Query to store data. React Query can handle the cache and its TTL.

When we send a message the chat is "mutating" and the chat that is in cache is not already the same that it actually is. In order to prevent the cache being not fresh, on success we update the chat in the cache, so the next time the user tries to fetch its cache, it won't as React Query has a cache, that is not stale, and is updated all the time the user sends/receives a message.